### DISCUSSION TITLE
Saving path for the yolo11n.pt model for AMP

### DISCUSSION THREAD

**Discussion by jeaho322** [ultralytics/ultralytics#19910] [Q&A]
Hi, I'm trying traning yolov8 model. because of my offline env, I can't use AMP (downloading yolo11n.pt). So I want to save it manually. what is the path of yolo11n.pt for using AMP?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @jeaho322, thank you for your interest in Ultralytics 🚀! We recommend checking out the [Docs](https://docs.ultralytics.com/) as a starting point, where you'll find detailed guides and examples for working with YOLO models, including [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage.

If this is a 🐛 Bug Report, please include a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug your issue.

If this is a custom training ❓ Question, please provide additional details, such as your training configuration, dataset details, and any relevant logs. Make sure to follow our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/) to optimize your setup.

For your offline environment, you can download the `yolo11n.pt` model manually from the official [Ultralytics Model Zoo](https://github.com/ultralytics/assets) and place it in your working directory or specify its path when initializing your model. 

Join the Ultralytics community to connect with others and find more support! For real-time chat, visit [Discord](https://discord.com/invite/ultralytics) 🎧. For detailed questions and discussions, explore [Discourse](https://community.ultralytics.com/). Or join the conversation on our [Subreddit](https://reddit.com/r/Ultralytics).

## Upgrade

It's always a good idea to ensure you're using the latest version of the `ultralytics` package with all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment and [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/). Update your setup using the following:

```bash
pip install -U ultralytics
```

## Environments

YOLO can be run in verified environments with pre-installed dependencies like [Python](https://www.python.org/), [PyTorch](https://pytorch.org/), [CUDA](https://developer.nvidia.com/cuda-zone), and [CUDNN](https://developer.nvidia.com/cudnn):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

This is an automated response to help guide you, but rest assured, an Ultralytics engineer will assist you soon! 😊

**Comment by glenn-jocher** [comment]
@jeaho322 for offline AMP usage with YOLO11, manually download [yolo11n.pt](https://github.com/ultralytics/assets/releases/download/v8.3.0/yolo11n.pt) and place it in your Ultralytics model cache directory at `~/.cache/ultralytics/hub/`. The training process will automatically detect this local copy.

**Reply by jeaho322** [reply]
thank you! but it's still not working. Is there another way what I can do? 
This directory didn't exist initially, so I created it and then did as you mentioned.

(base) XXXX@notebook-13490-0:~/.cache/ultralytics/hub$ ls
yolo11n.pt

AMP: checks skipped ⚠️. Offline and unable to download YOLO11n for AMP checks. Setting 'amp=True'. If you experience zero-mAP or NaN losses you can disable AMP with amp=False.

**Reply by glenn-jocher** [reply]
@jeaho322 for offline AMP validation, you can explicitly load the local model file by passing the full path to `YOLO()` like `YOLO('/path/to/yolo11n.pt')` in your training script. This bypasses automatic checks while maintaining AMP functionality. If issues persist, you can safely disable AMP with `amp=False` in your training arguments as suggested in the AMP checks documentation.

**Reply by Brunettow** [reply]
Hi, I just came across this post and noticed that the link doesn't seem to be working. Would you mind taking a look? Thank you in advance! 🙏

**Comment by Y-T-G** [comment]
You can place it in the working directory.
